cli/commands.go

```diff
@@ -16,7 +16,7 @@ var (
 		{
 			Name:      "update",
 			ShortName: "u",
-			Usage:     "",
+			Usage:     "Updates the collection, if no --collection flag provided, all collections will updated.",
```

Comment:
Update~~s~~ the collection. If no --collection flag provided, all collections will updated.